src/utils/htmlParser.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment, createElement } from 'react';
+import regex from './regex';
+
+const htmlParser = (html = '') => {
```

Comment:
I was confused by the name `htmlParser` at first. IMO `htmlStringToReact` would be more descriptive.

Reply:
Renamed as suggested.